paper-tab.html

```diff
@@ -169,8 +172,34 @@
 
           anchor.click();
         }
-      }
+      },
 
+      _focusedChanged: function(receivedFocusFromKeyboard) {
+        var ripple = this.getRipple();
+
+        if (this.focused && receivedFocusFromKeyboard) {
```

Comment:
Can you do something simpler like what we do for the InkyFocusBehavior? https://github.com/PolymerElements/paper-behaviors/blob/master/paper-inky-focus-behavior.html#L26